SharedPluginFiles/FormItPluginUI.js

```diff
@@ -87,7 +126,16 @@ FormIt.PluginUI.AlphaNumericInput = class AlphaNumericInput {
                 // ensure that only if the value is different than it was when we started, do we submit the function
                 if (event.currentTarget.value !== this.existingInputValue)
                 {
-                    this.submitTextFunction();
+                    const resultCallback = (returnVal) => {
```

Comment:
Line 129-138 and 153-161 looks like the same code.  If so, you can combine and move the code to above so it's not dup'd code.

Reply:
I'd like to do this, but I can't figure out how. This callback stuff is voodoo code to me, and wrapping it into a function has caused the plugins to stop working. I'm going to punt on this and refactor duplicate code later.